docs/benchmarks/data-caching.md

```diff
@@ -138,6 +138,6 @@ and the client on different sockets of the same machine
   [clientdocker]: https://github.com/parsa-epfl/cloudsuite/blob/master/benchmarks/data-caching/client/Dockerfile "Client Dockerfile"
```

Comment:
Can you also change paths like this to point to another branch other than master?
We will create a new branch with whatever we currently have as master as call it `CSv3`, so all these kind of paths should point to that branch.

Reply:
Let me directly push to a new branch called `CSv3`, alright?

Reply:
No not now.
We want to have these on the master branch and then we will do that.